libcodechecker/util.py

```diff
@@ -373,3 +373,17 @@ def get_line(file_name, line_no):
             return ''
     except IOError:
         return ''
+
+
+def get_user_input(msg):
```

Comment:
Can we start the server by hitting `Enter`? So not only for `y`/`yes`.

Reply:
I would keep this as it is. This feature changes/modifies data and the user should not accidentally do this.